src/utils/coinUtils.js

```diff
@@ -120,6 +120,17 @@ export const createTX = async ({
     setRBF = false
   }
 }: CreateTxOptions) => {
+  // Convert an address to the correct format that bcoin supports
```

Comment:
I've stared at this for 10 mins and can't seem to find how the code differs from before. Seems like you've just moved the call to `primitives.Address.fromString()` to inside of `toBcoinFormat`. But effectively this should produce the same logic. Did I miss something?

Reply:
Right, but this is not the only place `toBcoinFormat` gets called. It's the other use that's the problem.

Reply:
Ah. That code was hidden in Github. Needed to click to expand. Got it.